### PR TITLE
add script to convert Spherical3D data to netCDF

### PR DESCRIPTION
**Pull request by tukss**
The script contains rudimentary help string. The created files can be visualized with ParaView's netCDF Reader without modifications.

### CODE REVIEW

File: post_processing/sph3d_to_nc.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python
+import sys
```
Comment (cianwilson) [inline post_processing/sph3d_to_nc.py]: Is a copyright notice needed?

File: post_processing/sph3d_to_nc.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+
+def indexonly(f):
+    fsplit = os.path.basename(f).split('_')
+    if len(fsplit) > 1:
+        fsplit = fsplit[:-1]
+    return os.path.join(os.path.dirname(f), ''.join(fsplit))
+
+if args.all:
+    files = glob.glob('Spherical_3D/*_0*') + args.files
```
[the remark this answers is not in the document]
Comment (tukss) [reply post_processing/sph3d_to_nc.py]: Agreed. Doing it that way now.

**Comment by tukss** [comment]
> Should we consider standardizing the script names for these conversions? There's currently a `convert_full3d_to_vtu.py`.

I think standardized names are better. I renamed my script.

File: post_processing/rayleigh_diagnostics.py [hunk 3]
```diff
@@ -119,7 +119,7 @@ def __init__(self,filename,path='Spherical_3D/'):
           if index == "grid": continue
           fd = open(path+var_file, 'rb')
           self.vals[index] = swapread(fd,dtype='float64',count=nphi*ntheta*nr,swap=bs)
-
+          self.vals[index] = np.reshape(self.vals[index],(nphi,ntheta,nr), order = 'F')
```
[the remark this answers is not in the document]
Comment (tukss) [reply post_processing/rayleigh_diagnostics.py]: I changed `convert_full3d_to_vtu.py` accordingly.

File: post_processing/sph3d_to_nc.py (hunk 1 above)
Comment (tukss) [reply post_processing/sph3d_to_nc.py]: I added one.

**Comment by tukss** [comment]
Fixed a small bug.

**Pull request merged**